.github/workflows/basic-tests.yml

```diff
@@ -28,12 +28,9 @@ jobs:
     - name: preperation (generate code)
       run: |
         npm run build:shared:version
-    - name: Check Typescript Types
+    - name: Run all checks
       run: |
-        npm run check-types
-    - name: Run ESLint and Prettier
-      run: |
-        npm run lint
+        npm run check
```

Comment:
It's probably better if these scripts always run the higher level commands and the details can then be kept in `package.json`.